extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardModel.ts

```diff
@@ -94,6 +94,28 @@ export class BdcDashboardModel {
 	}
 }
 
+/**
+ * Retrieves the troubleshoot book URL for the specified service, defaulting to the BDC
+ * troubleshoot notebook if the service name is unknown.
+ * @param service The service name to get the troubleshoot notebook URL for
+ */
+export function getTroubleshootNotebookUrl(service: string): string {
+	switch (service) {
```

Comment:
Any chance casing changes could mess us up? Should we compare .ToLowerCase versions?